README.md

```diff
@@ -181,3 +183,36 @@ module.exports = {
 };
 
 ```
+
+### Using with Minio
+```
+hasura-backend-plus:
+   image: 'elitan/hasura-backend-plus'
+   restart: always
+   environment:
+     PORT: 3000
+     HASURA_GQE_ENDPOINT: 'https://<hasura-graphql-endpoint>'
```

Comment:
I think this can be:

`HASURA_GQE_ENDPOINT: http://graphql-engine:8080/v1alpha1/graphql`

This way this variable does not have to change when running the docker-compose locally or in production